homework.md

```diff
@@ -4,110 +4,111 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12,240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+using CSS helps you build responsive, future-proof, and highly accessible websites.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a text-based programming language used both on the client-side and server-side that allows you to make web pages interactive language used on the internet
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1-Remove a file from git without removing it from your file system. ...
+2-Undo local commits. ...
+3-Clean up local commits before pushing. ...
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+Iis producing a personal copy of someone else's projectt
 
-## Question 10 
+## Question 10
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+It means to duplicate and download everything in the repository
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+A Git log is a running record of commits
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+so that others may access them from the remote repository.
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+GitHub is a code hosting platform for version control and collaboration. It lets you and others work together on projects from anywhere.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+A commit represents some logical update to your collection of files. Imagine that you’re adding new items to your ideas lists, and you’ve added as many as you can think of. You’d like to capture that bit of work as a commit into your repository.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+codepen ,Visual Studio Code and GitHub Desktop
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+is used to save changes to a local repository after staging in Git
 
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+The Top bar there is a word "Help"
```

Comment:
Great call on the "Help" button Sarah. I didn't even think of that option. You might also want to include the "Git cheatsheet" and Slack in your answer though.

Reply:
Thanks, Bisioolajide for pointing that out I will update it.